nx-dev/nx-dev/lib/api.ts

```diff
@@ -34,13 +41,7 @@ export const nxDocumentsApi = new DocumentsApi({
     .filter((x) => !!x)
     .map((x) => convertToDocumentMetadata(x)),
   addAncestor: null,
-});
-export const nxRecipesApi = new DocumentsApi({
-  publicDocsRoot: 'nx-dev/nx-dev/public/documentation',
-  documentSources: [
-    documents.content.find((x) => x.id === 'nx-recipes'),
-  ].filter((x) => !!x) as DocumentMetadata[],
-  addAncestor: { id: 'recipes', name: 'Recipes' },
+  relatedSources: nxRecipesApi,
```

Comment:
Can we have this as an `Array()` of `DocumentsApi`? I think we will add more to the related resources option. That would let us do some kind of categorization as well.